src/app/components/timepicker/index.html

```diff
@@ -1,14 +1,162 @@
-<sky-demo-page
+<sky-tabbed-demo-page
   pageTitle="Timepicker"
-  skyuxModule="SkyTimepickerModule"
-  packageName="@skyux/datetime"
 >
 
-  <sky-demo-page-summary>
-    The timepicker module contains directives to let users select times. It is a SKY UX-themed replacement for the HTML <sky-code>input</sky-code> element with <sky-code>type="time"</sky-code>.
+<sky-demo-page-summary>
+    <p>The timepicker module contains directives to let users select times. It is a SKY UX-themed replacement for the HTML <sky-code>input</sky-code> element with <sky-code>type="time"</sky-code>.
     The value that users select is driven through the <sky-code>ngModel</sky-code> attribute
-    specified on the <sky-code>input</sky-code> element.
-  </sky-demo-page-summary>
+        specified on the <sky-code>input</sky-code> element.</p>
+</sky-demo-page-summary>
+    
+<sky-demo-page-example>
+    <sky-timepicker-demo></sky-timepicker-demo>
+</sky-demo-page-example>
+
+<sky-tabset id="content-tabs">
+
+    <sky-tab tabHeading="Design">
+
+        <sky-demo-page-content sectionHeading="Usage">
+        
+            <h3 class="sky-text-success">Use when</h3>
+            
+            <sky-docs-visual-example>
+              <sky-docs-visual-example-text>
+                  <p>Use timepicker when users need to specify a time of day.</p>
+              </sky-docs-visual-example-text>
+                <sky-docs-visual-example-image>
+                    <img src="~/assets/img/guidelines/timepicker/timepicker-usage-1.png" />
+                    <div class="caption caption-success sky-text-success">Do use timepicker when users need to input a 12-hour or 24-hour time.</div>
+              </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <sky-docs-visual-example>
+              <sky-docs-visual-example-text>
+                  <p>Use two timepickers when users need to specify a start time and stop time.</p>
+              </sky-docs-visual-example-text>
+              <sky-docs-visual-example-image>
+                    <img src="~/assets/img/guidelines/timepicker/timepicker-usage-2.png" />
```

Comment:
Start/end inputs should be 10px apart

Reply:
Good catch.

Reply:
do you think this example should be moved to the layout section?